nuxeo-common/src/main/java/org/nuxeo/common/utils/i18n/I18NUtils.java

```diff
@@ -64,4 +66,19 @@ public static String getMessageString(String bundleName, String key, Object[] pa
         return text;
     }
 
+    /**
+     * Parse a given locale string and return the matching Locale instance.
+     *
+     * @param locale
+     * @return
+     * @since 9.1
+     */
+    public static Locale parseLocale(String locale) {
+        StringTokenizer tokens = new StringTokenizer(locale, "-_");
```

Comment:
`StringTokenizer` is effectively a legacy class (see its Javadoc). Using `String.split` is preferred.